.aws/deploy/backend-task-definition.staging.json

```diff
@@ -0,0 +1,238 @@
+      "secrets": [
+        {
+          "name": "AUTH_TOKEN_EXPIRY_DURATION_IN_MILLISECONDS",
+          "valueFrom": "STAGING_AUTH_TOKEN_EXPIRY_DURATION_IN_MILLISECONDS"
+        },
+        {
+          "name": "AWS_BACKEND_EB_ENV_NAME",
+          "valueFrom": "STAGING_AWS_BACKEND_EB_ENV_NAME"
+        },
+        { "name": "AWS_REGION", "valueFrom": "STAGING_AWS_REGION" },
+        { "name": "CLIENT_ID", "valueFrom": "STAGING_CLIENT_ID" },
+        { "name": "CLIENT_SECRET", "valueFrom": "STAGING_CLIENT_SECRET" },
+        {
+          "name": "CLOUDMERSIVE_API_KEY",
+          "valueFrom": "STAGING_CLOUDMERSIVE_API_KEY"
+        },
+        { "name": "COOKIE_DOMAIN", "valueFrom": "STAGING_COOKIE_DOMAIN" },
+        { "name": "DB_ACQUIRE", "valueFrom": "STAGING_DB_ACQUIRE" },
+        { "name": "DB_MAX_POOL", "valueFrom": "STAGING_DB_MAX_POOL" },
+        { "name": "DB_MIN_POOL", "valueFrom": "STAGING_DB_MIN_POOL" },
+        { "name": "DB_TIMEOUT", "valueFrom": "STAGING_DB_TIMEOUT" },
+        { "name": "DB_URI", "valueFrom": "STAGING_DB_URI" },
+        {
+          "name": "DD_AGENT_MAJOR_VERSION",
+          "valueFrom": "STAGING_DD_AGENT_MAJOR_VERSION"
+        },
+        { "name": "DD_ENV", "valueFrom": "STAGING_DD_ENV" },
+        {
+          "name": "DD_LOGS_INJECTION",
+          "valueFrom": "STAGING_DD_LOGS_INJECTION"
+        },
+        { "name": "DD_SERVICE", "valueFrom": "STAGING_DD_SERVICE" },
+        { "name": "DD_TAGS", "valueFrom": "STAGING_DD_TAGS" },
+        {
+          "name": "DD_TRACE_STARTUP_LOGS",
+          "valueFrom": "STAGING_DD_TRACE_STARTUP_LOGS"
+        },
+        {
+          "name": "E2E_TEST_GH_TOKEN",
+          "valueFrom": "STAGING_E2E_TEST_GH_TOKEN"
+        },
+        { "name": "E2E_TEST_REPO", "valueFrom": "STAGING_E2E_TEST_REPO" },
+        { "name": "E2E_TEST_SECRET", "valueFrom": "STAGING_E2E_TEST_SECRET" },
+        { "name": "EFS_VOL_PATH", "valueFrom": "STAGING_EFS_VOL_PATH" },
+        {
+          "name": "ENCRYPTION_SECRET",
+          "valueFrom": "STAGING_ENCRYPTION_SECRET"
+        },
+        {
+          "name": "FF_DEPRECATE_SITE_QUEUES",
+          "valueFrom": "STAGING_FF_DEPRECATE_SITE_QUEUES"
+        },
+        { "name": "FRONTEND_URL", "valueFrom": "STAGING_FRONTEND_URL" },
+        {
+          "name": "GGS_EXPERIMENTAL_TRACKING_SITES",
+          "valueFrom": "STAGING_GGS_EXPERIMENTAL_TRACKING_SITES"
+        },
+        {
+          "name": "GITHUB_BUILD_ORG_NAME",
+          "valueFrom": "STAGING_GITHUB_BUILD_ORG_NAME"
+        },
+        {
+          "name": "GITHUB_BUILD_REPO_NAME",
+          "valueFrom": "STAGING_GITHUB_BUILD_REPO_NAME"
+        },
+        { "name": "GITHUB_ORG_NAME", "valueFrom": "STAGING_GITHUB_ORG_NAME" },
+        {
+          "name": "GROWTHBOOK_CLIENT_KEY",
+          "valueFrom": "STAGING_GROWTHBOOK_CLIENT_KEY"
+        },
+        {
+          "name": "INCOMING_QUEUE_URL",
+          "valueFrom": "STAGING_INCOMING_QUEUE_URL"
```

Comment:
how are the secrets injected?

Reply:
ssm, the role just needs to have access to efs.

Reply:
https://repost.aws/knowledge-center/ecs-data-security-container-task

Reply:
sweet